static/i18n/en/app.json

```diff
@@ -375,7 +375,9 @@
         },
         "pending": {
           "title": "Broadcasting transaction..."
-        }
+        },
+        "broadcastError":
+          "Transaction may have failed. Please check back in a few minutes to make sure your transaction did not go through"
```

Comment:
Your transaction may have failed. Please check back in a few minutes to make sure your transaction did not go through before trying again.